### ISSUE TITLE
Highlight hovering not working

### ISSUE THREAD

**Issue by VineethaVi** [Bridgeconn/VachanOnline-v2#469]
Description - In progress

**Comment by AnnCherianA** [comment]
Identified during sanity  test 
@VineethaVi please share  your finding with complete description with proper screenshots and mention the device as well.

**Comment by AnnCherianA** [comment]
Please upload a screen recording of your observation.

**Comment by AnnCherianA** [comment]
Tested on the website. 
Highlight feature does not work as expected.  On hovering over a highlighted verse, the highlight (color)gets disabled and only the hover works ie the verse is highlighted in grey color. 
The use of the highlight feature itself is to  make a verse  stand out to easily identify the verse later. 
Attaching the screenshots 


![Image](https://github.com/Bridgeconn/VachanOnline-v2/assets/74647279/4b0c992a-ce56-4167-b2c8-2275b604b82b)



![Image](https://github.com/Bridgeconn/VachanOnline-v2/assets/74647279/1f70e68d-5153-4ca6-aebd-8080758b2b57)

**Comment by VineethaVi** [comment]
> Please upload a screen recording of your observation.

Attaching the screen recording 

https://github.com/Bridgeconn/VachanOnline-v2/assets/132653333/aa9eb970-f5bc-4439-9ed3-4d5885e3d778

**Comment by AnnCherianA** [comment]
@VineethaVi Can you confirm if this happens still

**Comment by VineethaVi** [comment]
> @VineethaVi Can you confirm if this happens still

@AnnCherianA  It is not happening now. The highlights are working properly. For highlighting the first selected verse,  no need to select any other verses .

**Comment by VineethaVi** [comment]
@AnnCherianA  can I move to done ?

**Comment by AnnCherianA** [comment]
Yes, you can test on the updated  staging site  and update the result here. It may have got fixed in the process

**Comment by VineethaVi** [comment]
Testing is passed. The highlights are working properly.
Testing done on chrome.

Adding the screenshot of website

![highlightwebsite](https://github.com/Bridgeconn/VachanOnline-v2/assets/132653333/71fcbd03-b7e1-4dee-a847-03bcdf8e1ed2)



Adding the screenshot of mobile browser


![highlightmobile](https://github.com/Bridgeconn/VachanOnline-v2/assets/132653333/ba47aed0-3375-42cd-894a-660d1afae863)

**Comment by AnnCherianA** [comment]
@VineethaVi  did you test with toggle on or off ? 
 I tested during sprint 2 and there was no toggle  button for highlight then.   Hovering over a highlighted verse would disable the highlight. Now there is a toggle available. Please test highlight with all its related settings and share the screenshot for highlight toggle on

**Comment by VineethaVi** [comment]
After turn on the toggle button ,the highlighted verse shows in grey color when we hover that verse.

Adding the screenshot of website

![toggle on highlight](https://github.com/Bridgeconn/VachanOnline-v2/assets/132653333/49eb1541-c618-439e-9334-fe8fc9efc285)


Adding the screenshot of mobile browser


![mobilehighlight](https://github.com/Bridgeconn/VachanOnline-v2/assets/132653333/1227ecf2-5252-45bb-852f-9f3ad0ade43b)

**Comment by VineethaVi** [comment]
moving to done
